superset-frontend/cypress-base/cypress/support/index.d.ts

```diff
@@ -41,6 +41,7 @@ declare namespace Cypress {
     cleanDashboards(): cy;
     loadChartFixtures(): cy;
     loadDashboardFixtures(): cy;
+    setConsoleMessages(consoleMessages: (string | RegExp)[]): cy;
```

Comment:
Maybe `allowConsoleMessages` would be a better name?

Reply:
Agreed, done!